src/components/Form.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook } from '../redux/books/bookSlice';
 import './book.css';
 
 export default function Form() {
+  const dispatch = useDispatch();
+  const books = useSelector((store) => store.books);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const title = e.target[0].value;
+    const author = e.target[1].value;
```

Comment:
- Kindly, use the local state for `title` and `author`, which is updated through the onChange event 👍
```
 const [title, setTitle] = useState("")
 const [author, setAuthor] = useState("")
```